quizzes/rules.css

```diff
@@ -0,0 +1,29 @@
+/*** Style-for-rules ***/
+.rule-container {
+  width: 50%;
+  margin: var(--xxl-space) auto;
+  padding: var(--lg-space);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: var(--xl-space);
+  border: 2px solid var(--primary-color);
```

Comment:
`rem` could be used here

Reply:
Actually I'm changing the root font size in media queries and I don't really want the border to change accordingly. So that's why I used px instead of rem.